cmd/crio/status.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/cri-o/cri-o/internal/client"
+
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	defaultSocket = "/var/run/crio/crio.sock"
+	idArg         = "id"
+	socketArg     = "socket"
+)
+
+var statusCommand = &cli.Command{
```

Comment:
I just had the idea: can we reuse this code in `cmd/crio-status/main.go`? If we make this variable public, we could import it there, and update the Usage and error returned with the deprecation warning. then, if we happen to have a change required, we only need to change it in one spot

Reply:
Yes, we could do that, if we are certain that we will only have a single binary `crio` going forward.

If not, a better option would be to move `cli.Command` instances like `config`, `publish`, `version`, etc to a common folder `cli` next to the `cmd` folder, make them public, and have the both `crio` and `crio-status` import and use them. This way if we ever need to add more binaries, we won't need to refactor code and new binaries can just import common commands and use them.

Let me know your thoughts and I can change the PR accordingly.

Reply:
I would say it's most likely to say as a sub-command of `crio`. If we really need to, we can move it later, but I think it will be under `crio` for a while

Reply:
Since the current status command is part of the `main` package and it cannot be imported into another `mail` package. So I've move the cli commands to the internal `criocli` package and imported them to the binary cli's.